packages/ui/cypress/tests/network-switch.cy.ts

```diff
@@ -0,0 +1,58 @@
+import { accountDisplay } from '../support/page-objects/components/accountDisplay'
+import { settingsPageWatchAccountUrl } from '../fixtures/landingData'
+import { settingsPage } from '../support/page-objects/settingsPage'
+import { topMenuItems } from '../support/page-objects/topMenuItems'
+import { watchMultisigs } from '../fixtures/watchAccounts/watchMultisigs'
+import convertRococoAddressToKusama from '../utils/convertRococoAddressToKusama'
+
+const {
+  publicKey: multisigPublicKey,
+  address: multisigAddress,
+  pureAddress: multisigPureAddress,
+  name: multisigName
+} = watchMultisigs['multisig-with-pure']
+
+describe('Network can be switched', () => {
+  it('should switch account using menu', () => {
```

Comment:
I'm not sure I understand the purpose of this title here in the "network switch" test. Testing the account switch is something that is either already done elsewhere, or should be done inside an account related test. It seems out of place.

Reply:
I think it's a mistake, it should be should switch network